plugins/modules/source_control/git/git_commit.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Federico Olivieri (lvrfrc87@gmail.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: git_commit
+author:
+    - "Federico Olivieri (@Federico87)"
+version_added: "2.10"
+short_description: Perform git add and git commit operations.
+description:
+    - Manage C(git add) and C(git commit) on a local git.
+options:
+    path:
+        description:
+            - Folder path where C(.git/) is located.
+        required: true
+        type: path
+    comment:
+        description:
+            - Git commit comment. Same as C(git commit -m).
+        type: str
+        required: true
+    add:
+        description:
+            - List of files under C(path) to be staged. Same as C(git add .).
+              File globs not accepted, such as C(./*) or C(*).
+        type: list
+        elements: str
+    empty_commit:
+        description:
+            - Drive module behaviour in case nothing to commit.
+              If C(allow) empty commit is allowed, same as C(--allow-empty).
+              Do not commit if C(skip). Fail job if C(fail).
+        choices: ['allow', 'fail', 'skip']
+        type: str
+        default: 'fail'
+requirements:
+    - git>=2.10.0 (the command line tool)
+'''
+
+EXAMPLES = '''
+- name: Add and commit two files.
+  community.general.git_commit:
+    path: /Users/federicoolivieri/git/git_test_module
+    comment: My amazing backup
+    add: ['test.txt', 'txt.test']
+    empty_commit: fail
+
+- name: Empty commit.
+  community.general.git_commit:
+    path: /Users/federicoolivieri/git/git_test_module
+    comment: My amazing empty commit
+    empty_commit: allow
+    add: ['.']
+
+- name: Skip if nothing to commit.
+  community.general.git_commit:
+    path: /Users/federicoolivieri/git/git_test_module
+    comment: Skip my amazing empty commit
+    empty_commit: skyp
+    add: ['.']
+'''
+
+RETURN = '''
+output:
+    description: list of git cli command stdout
+    type: list
+    returned: always
+    sample: [
+        "[master 99830f4] Remove [ test.txt, tax.txt ]\n 4 files changed, 26 insertions(+)..."
+    ]
+'''
+
+import os
+from ansible.module_utils.basic import AnsibleModule
+from ansible.utils.display import Display
+
+display = Display()
+
+
+def git_add(module):
+
+    add = module.params.get('add')
+    path = module.params.get('path')
+
+    if not add:
+        add = ["."]
+    # "git add do_not_exist" -> rc: 128
+    # "git add i_exist"      -> rc: 0
+    # "git add ."            -> rc: 0
+    add_cmds = [
+        'git',
+        'add',
+    ]
+    for item in add:
+        add_cmds.insert(len(add_cmds), item)
```

Comment:
```suggestion
        add_cmds.append(item)
```
Or instead of the `for` loop:
```.py
add_cmds.extend(add)
```
This will add all elements from `add` to `add_cmds`.